### ISSUE TITLE
[Demo] Site Theme keeps breaking and rendering HTML only

### ISSUE THREAD

**Issue by tonyzeoli** [netmix/radio-station#211]
**Describe the bug**

Not sure why this is happening, but the theme for the demo site keeps breaking. The theme reverts to HTML. I keep updating the plugins and dumping the cache, but the issue keeps returning.

![Screen Shot 2020-05-06 at 7 18 32 PM](https://user-images.githubusercontent.com/4053819/81237918-c63ad200-8fce-11ea-9f33-9799ca7a3956.png)

**To Reproduce**
1. Go to radiostationdemo.com
2. View home page theme is broke, renders only as HTML

**Expected behavior**

Theme is supposed to load properly.

**Screenshots**

![Screen Shot 2020-05-06 at 7 18 32 PM](https://user-images.githubusercontent.com/4053819/81238002-f08c8f80-8fce-11ea-95b2-084cfd128ece.png)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Catalina 10.15.4
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22] 81.0.4044.129 (Official Build) (64-bit)

**Additional context**

**Comment by tonyzeoli** [comment]
@majick777 I dumped the cache again to bring back the theme and that worked. But it comes and goes.

**Comment by majick777** [comment]
@tonyzeoli  I also had to clear the autoptimize cache the other day - for the Show page to display properly, as it's CSS wasn't loading... It does seem to be fine now though, but if it plays up again I think that it probably the issue.

**Comment by andyrb412** [comment]
I've tested the site in several browsers.  Loads fine in Safari, Edge and Chrome.

Failed to load in Firefox.  Console shows:

<img width="862" alt="Screen Shot 2020-05-07 at 21 40 58" src="https://user-images.githubusercontent.com/18440817/81290490-8269ca80-90ab-11ea-967c-deb830a60485.png">

It appears there's an issue with loading necessary scripts, which is resulting in a fatal error (and CSS not loading).  Refreshing the page sees it load normally, but fails.

I'm a little unsure why it's periodically failing though, because if Firefox is blocking the script it should be all the time, not intermittently.

**Comment by tonyzeoli** [comment]
@andyrb412 Thanks or checking that.

@majick777 Can you let me know if the Firefox fix you implemented solves the issue here?

**Comment by majick777** [comment]
@tonyzeoli I added wpauthbox.js to the Autoptimize exclude scripts field and cleared the cache. This seems to have fixed the first console error in the screenshot. The second looks like gravity forms so I disabled that as not used on the Demo site anyway. I'm not sure there's anything further to do here unless something else comes up again, eg. the stylesheets not loading properly. But that can be caused by a temporary network glitch and can happen on any website.
